src/evict/evict_page.c

```diff
@@ -95,15 +178,15 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
     WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
     WT_PAGE *page;
-    uint64_t eviction_time, eviction_time_seconds;
-    bool clean_page, closing, force_evict_hs, inmem_split, local_gen, tree_dead;
+    uint8_t stats_flags;
+    bool clean_page, closing, inmem_split, tree_dead, local_gen;
```

Comment:
`tree_dead` and `local_gen` need re-alphabetising here

Reply:
updated here! : [6e0e9b2](https://github.com/wiredtiger/wiredtiger/pull/9333/commits/6e0e9b200cfeb03f06c774fac79ea5cb60dc7770)